corehq/apps/reports/view_helpers.py

```diff
@@ -13,6 +13,8 @@
 from corehq.apps.hqwebapp.templatetags.proptable_tags import get_display_data
 from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
 
+MAX_CHILD_CASES = 100
```

Comment:
How did you pick ```100```?

Reply:
it's arbitrary - this is enough of an edge-case that I don't think it's yet worth building out full support like pagination for it, so I wanted a number big enough that most projects will never need to know there's a limit.

Reply:
👍